src/libstd/io/stdio.rs

```diff
@@ -893,9 +893,9 @@ impl fmt::Debug for StderrLock<'_> {
 /// use std::io::prompt;
 ///
 /// fn main() {
-///     let user_input = prompt("Please enter some text: ");
+///     let name = prompt("What is your name?: ");
```

Comment:
The question mark and colon looks conflicting.

Reply:
fixed the confliction.